responders/src/UI/embc-responder/src/app/feature-components/wizard/step-create-profile/step-create-profile.component.html

```diff
@@ -22,7 +22,7 @@
         'tab-margin custom-tab-link': tabItem.name !== 'review'
       }"
     >
-      <span class="span-height" (click)="isAllowed(tabItem.name, $event)"
+      <span class="span-height" (click)="isAllowed(tabItem.route, $event)"
```

Comment:
'route' is the incorrect property to be used here. The correct property that should be used is 'name'